src/gluonts/mx/trainer/_base.py

```diff
@@ -120,10 +134,8 @@ def __init__(
         weight_decay: float = 1e-8,
         init: Union[str, mx.initializer.Initializer] = "xavier",
         hybridize: bool = True,
-        avg_strategy: Union[
-            AveragingStrategy, IterationAveragingStrategy
-        ] = SelectNBestMean(num_models=1),
-        post_initialize_cb: Optional[Callable[[mx.gluon.Block], None]] = None,
+        callbacks: Union[Callback, List[Callback]] = CallbackList([]),
```

Comment:
I don't like this. This should always be the list, not either or.

Reply:
always `List[Callback]` you mean? with default `[]`

Reply:
Yes.